web-registry/src/pages/Marketplace/Storefront/hooks/useNormalizedSellOrders.tsx

```diff
@@ -0,0 +1,169 @@
+import { useMemo, useState } from 'react';
+import {
+  ApolloClient,
+  NormalizedCacheObject,
+  useApolloClient,
+} from '@apollo/client';
+import { useQueries, useQuery } from '@tanstack/react-query';
+
+import {
+  SortCallbacksType,
+  TablePaginationParams,
+} from 'web-components/lib/components/table/ActionsTable';
+import { DEFAULT_ROWS_PER_PAGE } from 'web-components/src/components/table/Table.constants';
+
+import { useLedger } from 'ledger';
+import { getSimplePriceQuery } from 'lib/queries/react-query/coingecko/simplePrice/simplePriceQuery';
+import { getBatchQuery } from 'lib/queries/react-query/ecocredit/getBatchQuery/getBatchQuery';
+import { getProjectsQuery } from 'lib/queries/react-query/ecocredit/getProjectsQuery/getProjectsQuery';
+import { getMetadataQuery } from 'lib/queries/react-query/registry-server/getMetadataQuery/getMetadataQuery';
+import { getAllProjectsQuery } from 'lib/queries/react-query/registry-server/graphql/getAllProjectsQuery/getAllProjectsQuery';
+import { getAllCreditClassesQuery } from 'lib/queries/react-query/sanity/getAllCreditClassesQuery/getAllCreditClassesQuery';
+
+import { useFetchSellOrders } from 'features/marketplace/BuySellOrderFlow/hooks/useFetchSellOrders';
+import { normalizeToUISellOrderInfo } from 'pages/Projects/hooks/useProjectsSellOrders.utils';
+import { UISellOrderInfo } from 'pages/Projects/Projects.types';
+import { SellOrderInfoExtented } from 'hooks/useQuerySellOrders';
+
+import { client as sanityClient } from '../../../../lib/clients/sanity';
```

Comment:
should be able to use absolute path here too
```suggestion
import { client as sanityClient } from 'lib/clients/sanity’;
```

Reply:
Not sure why the quote changed in the diff - maybe dont’ commit this suggestion directly